.gitignore

```diff
@@ -1,3 +1,3 @@
-node_modules
+/node_modules
```

Comment:
but, in fact I not tested without test/foo/node_modules to see if the same behavior works... I'll look on it.